### PR TITLE
Inline entity in external schema

### PR DESCRIPTION
**Pull request by Cypher1**
Tests some edge cases in parsing and type checking schemas containing inline schemas.

CC: @piotrswigon

### CODE REVIEW

File: src/runtime/manifest.ts [hunk 1]
```diff
@@ -465,7 +465,8 @@ export class Manifest {
         } else {
           preamble = `Post-parse processing ${severity} caused by`;
         }
-        message = `${preamble} '${fileName}' line ${e.location.start.line}.
+        const fileNameStr = fileName ? `'${fileName}' ` : '';
```
Comment (csilvestrini) [inline src/runtime/manifest.ts:468]: Just a thought, would it be clearer to report an error like this instead?

```
Post-parse processing error caused by <no filename> line 7.
```

**Review by csilvestrini — APPROVED** [review]
A tip for next time: it's generally good to keep separate changes in separate PRs. The `manifest.ts` and `manifest-test.ts` changes look like they're easily separable from the `schema-test.ts` changes. 

Having smaller CLs makes them easier to review, makes it easier for the person assembling changelogs, and helps you out if one of them needs to be rolled back for some reason.

File: src/runtime/manifest.ts (hunk 1 above)
Comment (Cypher1) [reply src/runtime/manifest.ts:468]: Split into https://github.com/PolymerLabs/arcs/pull/6616

Had them as different commits just generally try to avoid opening CLs that have such small changes (because of the overhead of copybara, the bots and code review)
